### PR TITLE
user adaptation for jobs API

### PR DESCRIPTION
**Pull request by yuritk**
Included: usertype (default: "hirer"), displayname, description

### CODE REVIEW

File: src/mutation/RegisterEmailMutation.js [hunk 1]
```diff
@@ -17,8 +17,14 @@ export default mutationWithClientMutationId({
     password: {
       type: new GraphQLNonNull(GraphQLString),
     },
+    displayname: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    description: {
+      type: new GraphQLNonNull(GraphQLString),
```
Comment (jgcmarins) [inline src/mutation/RegisterEmailMutation.js]: You don't need `GraphQLNonNull` here. See this: https://medium.com/@calebmer/when-to-use-graphql-non-null-fields-4059337f6fc8

**Comment by yuritk** [comment]
yarn run test
-- LoginEmailMutation: description required, why? Input fields is correct both on code and spec.js

**Comment by jgcmarins** [comment]
weird.
I did not know about this.
we can send a PR here: https://github.com/entria/graphql-dataloader-boilerplate
to fix

**Comment by yuritk** [comment]
Found, my mistake!

It turns out that I have not updated some "new users"

**Pull request merged**